vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java

```diff
@@ -119,7 +120,7 @@ private Object findByKey(String key) {
 			return value;
 		}
 
-		value = Config.get(request.getSession(), key);
+		value = Config.get(request.getSession(false), key);
```

Comment:
@cristianoperez thanks! it sounds better as a default value.

I'd only keep a fallback strategy for users that for some reason want a new session to be created. 
what do you think?

for example:
```
request.getSession(createNewSession())

//...
protected boolean createNewSession(){
    return false;
}
```

what do you think?

Reply:
done